docs/preview_conversion_analyzer.md

```diff
@@ -0,0 +1,172 @@
+# Examples
+
+#### Namespaces Before Running Analyzers:
+```csharp
+using Windows.UI;
+using Windows.UI.ViewManagement;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+```
+
+#### Namespaces After Running Analyzers:
+```csharp
+using Microsoft.UI;
+using Microsoft.UI.ViewManagement;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls;
+```
+
+#### OnLaunched Method Before Running Analyzers:
+```c#
+protected override async void OnLaunched(LaunchActivatedEventArgs args)
+{
+    await EnsureWindow(args)
+}
+```
+
+#### OnLaunched Method After Running Analyzers:
+```csharp
+protected override async void OnLaunched(Microsoft.UI.Xaml.LaunchActivatedEventArgs args)
+{
+    await EnsureWindow(args.UWPLaunchActivatedEventArgs)
+}
+```
+
+#### .Net Projections Before Running Analyzers:
+```csharp
+public sealed partial class CommandBarPage : INotifyPropertyChanged
+```
+
+#### .Net Projections After Running Analyzers:
+```csharp
+ public sealed partial class CommandBarPage : Microsoft.UI.Xaml.Data.INotifyPropertyChanged
+```
+
+#### .Net Struct Constructor Before Running Analyzers:
+```csharp
+CornerRadius c1 = new CornerRadius(4);
+CornerRadius c2 = new CornerRadius(4, 2, 2 4);
+```
+
+#### .Net Struct Constructor After Running Analyzers:
+```csharp
+CornerRadius c1  = CornerRadiusHelper.FromUniformRadius(4);
+CornerRadius c2  = CornerRadiusHelper.FromRadii(4, 2, 2, 4);
+```
+
+# Conversion Process
+
+**Note: `Microsoft.WinUI.Convert` NuGet Package is Currently Unavailable During Prerelease**
+
+**It is recommended to use the  [try-convert](https://github.com/dotnet/try-convert/blob/feature/winui/WinUIConvert.md) tool for the full conversion process.**
+
+Necessary steps for converting a WinUI C# App to WinUI3: 
+
+1. Install `Microsoft.WinUI` and `Microsoft.WinUI.Convert` NuGet packages in your app using the NuGet package manager: see the [Install WinUI 3 Preview](https://docs.microsoft.com/en-us/windows/apps/winui/winui3/#install-winui-3-preview-2) page for more information.
+    
+    ![Visual Studio Lightbulb Suggestion](./images/newPackage.png#thumb)
+
+2. If installed, uninstall the Microsoft.UI.Xaml NuGet package from your solution. Additional conflicting packages such as `Microsoft.Xaml.Behaviors.*` may also need to be removed.
+    
+    ![Visual Studio Lightbulb Suggestion](./images/uninstallOldPackage.png#thumb)
+
+3. Light Bulb suggestions should highlight issues that need to be updated for WinUI3 conversion. 
+
+    ![Visual Studio Lightbulb Suggestion](./images/lightbulb.png)
+
+4. Click the down arrow by the lightbulb, Convert to WinUI3 and select Fix all occurences in project. 
+
+    ![Visual Studio Lightbulb Suggestion](./images/analyzer-1.png)
+
+5. All namespace changes should be fixed in your project!
+
+6. The Analyzer can only parse C# code and not Xaml. WebView is now WebView2 and should be updated: see the [Getting Started](https://docs.microsoft.com/en-us/windows/apps/winui/winui3/) page for more information.
```

Comment:
not sure if this is necessary, it's implied in the "what this doesn't do bit". if anything, you can go into more detail there that there are differences which can't be converted

Reply:
Removed this note